cloudinit/cmd/main.py

```diff
@@ -745,8 +745,12 @@ def status_wrapper(name, args):
     else:
         try:
             status = json.loads(util.load_text_file(status_path))
-        except Exception:
-            pass
+        except OSError:
```

Comment:
If this doesn't happen, then either a previous stage failed to create a status.json, or something removed/modified it/corrupted it, or a cloud-init stage is getting manually run out of order([basically the same thing as this PR](https://github.com/canonical/cloud-init/pull/5209)).

All of these sound like "unexpected / bad things" to me - wouldn't a warning make sense?

Reply:
@TheRealFalcon waiting for response on this comment

Reply:
but that other PR already contains a warning, right? I guess it seems like fairly large shift from "it doesn't matter if this fails because we'll create it anyway" to "we need to warn you about this even though it has no bearing on the resulting artifact".

Reply:
That's fair, I'll remove it.